vllm/model_executor/models/ultravox.py

```diff
@@ -301,7 +302,15 @@ def forward(
 @MULTIMODAL_REGISTRY.register_max_multimodal_tokens(
     "audio", get_ultravox_max_audio_tokens)
 @MULTIMODAL_REGISTRY.register_processor(UltravoxMultiModalProcessor)
-class UltravoxModel(nn.Module, SupportsMultiModal, SupportsPP):
+class UltravoxModel(nn.Module, SupportsMultiModal, SupportsPP, SupportsLoRA):
+    packed_modules_mapping = {}
+    # llama3 supported modules
+    supported_lora_modules = [
+        'up_proj', 'down_proj', 'gate_proj', 'v_proj', 'o_proj', 'k_proj',
+        'q_proj'
+    ]
+    embedding_modules = {}
```

Comment:
Is `llama3` the language backbone of `UltravoxModel`, or are there variants that use other LLMs?

Reply:
Mostly it uses llama but also use Mistral in some cases